zeebe/gateway-rest/pom.xml

```diff
@@ -231,7 +230,7 @@
               <generatorName>spring</generatorName>
               <inputSpec>${openapi.dir}/rest-api.yaml</inputSpec>
               <modelPackage>io.camunda.zeebe.gateway.protocol.rest</modelPackage>
-              <schemaMappings>ProblemDetail=org.springframework.http.ProblemDetail</schemaMappings>
+              <typeMappings>OffsetDateTime=String</typeMappings>
```

Comment:
:question: Do we need to add this property in the java client as well?

Reply:
Currently, we don't need to. I am not using the date attributes of the resulting model class for user task updates from the client directly. Instead, I am setting String properties on the map of changes for the request. If we'd used those date attributes directly, we would have had to either adjust our implementation or add something like this to the client POM as well.

Reply:
For example, I am doing this in the client
```java
request.getChangeset().put(Changeset.JSON_PROPERTY_FOLLOW_UP_DATE, "");
```
where the `request.getChangeset()` is generated to be a subclass of a `HashMap` in the client.